engine/resource/src/async/load_queue_sync.cpp

```diff
@@ -73,17 +73,7 @@ namespace dmLoadQueue
             return RESULT_INVALID_PARAM;
         }
 
-        if (request->m_PreloadInfo.m_LoadResourceFunction)
-        {
-            load_result->m_LoadResult = request->m_PreloadInfo.m_LoadResourceFunction(queue->m_Factory, request->m_CanonicalPath, request->m_Name, size, &queue->m_LoadBuffer, request->m_PreloadInfo.m_Context);
-            *buf = queue->m_LoadBuffer.Begin();
-            assert(queue->m_LoadBuffer.Size() == *size);
-        }
-        else
-        {
-            load_result->m_LoadResult = dmResource::LoadResource(queue->m_Factory, request->m_CanonicalPath, request->m_Name, buf, size);
-        }
-
+        load_result->m_LoadResult    = dmResource::LoadResource(queue->m_Factory, request->m_CanonicalPath, request->m_Name, buf, size);
```

Comment:
rolled back some of the changes from #8416 since we are not using the "load queue" anymore in sys.load_buffer_async